project_generator/commands/build.py

```diff
@@ -24,16 +24,11 @@
 
 def run(args):
     # Export if we know how, otherwise return
```

Comment:
Just a note, seems like it should be still possible - With this build, previously was possible to use pgen for non-pgen projects. As -f would be a project file (.uvproj or .ewp), which then pgen would invoke and build . And user would get all this nice (TO DO) messages like 0 warnings, 4 errors.